fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -4079,6 +4075,14 @@ Reference<TCTeamInfo> DDTeamCollection::buildLargeTeam(int teamSize) {
 		}
 	}
 
+	if (totalShardCount >= SERVER_KNOBS->DD_MAXIMUM_LARGE_TEAMS) {
```

Comment:
Would it make sense to rename the knob to prevent confusion going forward? Maybe `DD_MAX_SHARDS_ON_LARGE_TEAMS`? If that's too long, at the very least we should update the comment at the knob definition clarifying the new meaning.